multi_project/core/build.gradle

```diff
@@ -0,0 +1,18 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ *
+ * This project uses @Incubating APIs which are subject to change.
+ */
+
+plugins {
+    id 'multi_project.java-application-conventions'
+}
+
+dependencies {
+    implementation files('customlib/utils-1.3.5.jar')
+}
+
+application {
+    // Define the main class for the application.
+    mainClass = 'multi_project.api.App'
```

Comment:
Why do we need this definition inside `core` module?

Reply:
idk, gradle init created it, project works wout it, so i remove him